wiki/Project_Loved/ru.md

```diff
@@ -0,0 +1,46 @@
+# Project Loved
+
+**Project Loved** - это платформа для голосования, которая продвигает карты в категорию [Любимые](/wiki/Beatmaps#loved). Проект ведёт [Команда Project Loved](/wiki/People/Project_Loved_Team).
+
+## Продвижение карты в «Любимые»
+
+Карты получают статус «Любимые» путём номинирования и голосования.
+
+Члены Project Loved Team, называемые **капитанами**, регулярно номинируют карты для различных игровых модов. Почти всегда карты берутся из сообщества osu! Актуальные формы и списки можно найти по следующим ссылкам:
+
+- [Форма для osu!/osu!taiko/osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [Форма для osu!mania](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
+- [Список osu!/osu!taiko/osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
+- [Список osu!mania](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit)
+
+*Примечание: Если у вас ограничен доступ к Гугл Формам, вы можете связаться с любым [Участником Команды Project Loved](/wiki/People/Project_Loved_Team#team-members), что бы карту представили от вашего имени.*
+
+Затем номинированые карты будут выложены на [форумe Project Loved](https://osu.ppy.sh/community/forums/120), где участники сообщества могут поголосовать, хотят ли они видеть карту в категории «Любимые». Карты, достигшие определенного процента голосов «Да» в момент окончания голосования, будут перемещены в категорию «Любимые».
+
+### Приемлемость
+
+Еще есть несколько небольших критериев, которые должны быть соблюдены, что бы карта была номинирована. Капитаны считают карту подходящей для категории «Любимые», если:
+
+- Она не находится в категориях [Квалицированные](/wiki/Beatmaps#qualified), [Рейтинговые](/wiki/Beatmaps#ranked) или [Одобренные](/wiki/Beatmaps#approved)
+- Создатель карты не против того, что бы его карта попала в категорию «Любимые»
+- Создатель карты не забанен или не имеет рестрикт.
+- Как минимум одна сложность может быть пройдена.
+- Карта как минимум должна быть добавлена в избранные 30 раз. (Относиться только к osu!standard картам)
+
+Тем не менее, несмотря на то, что почти все ["Ожидащие" и "В разработке"](/wiki/Beatmaps#pending) карты соответствуют этим критериям, не все попадают в категорию «Любимые», из-за более избирательных требований к номинации и голосованию.
+
+Карты, входящие в категорию «Любимые» должны следовать еще нескольким наборам правил:
+
+- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Song Content Rules](/wiki/Rules/Song_Content_Rules)
+- [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations)
```

Comment:
you can use the Russian names of these articles/sections, if they exist. also for line 34 the rest of the language ("The", "section of the") should be translated

Reply:
Done